exchanges/bitstamp/bitstamp_wrapper.go

```diff
@@ -315,6 +318,38 @@ func (b *Bitstamp) UpdateTradablePairs(ctx context.Context, forceUpdate bool) er
 	return b.EnsureOnePairEnabled()
 }
 
+// UpdateOrderExecutionLimits sets exchange execution order limits for an asset type
+func (b *Bitstamp) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) error {
+	if a != asset.Spot {
+		return common.ErrNotYetImplemented
+	}
+	symbols, err := b.GetTradingPairs(ctx)
+	if err != nil {
+		return err
+	}
+	limits := make([]order.MinMaxLevel, 0, len(symbols))
+	for x, info := range symbols {
+		if symbols[x].Trading != "Enabled" {
+			continue
+		}
+		pair, err := currency.NewPairFromString(symbols[x].Name)
+		if err != nil {
+			return err
+		}
+		limits = append(limits, order.MinMaxLevel{
+			Asset:                   a,
+			Pair:                    pair,
+			PriceStepIncrementSize:  1 / math.Pow10(info.CounterDecimals),
+			AmountStepIncrementSize: 1 / math.Pow10(info.BaseDecimals),
+			MinimumBaseAmount:       info.MinimumOrder,
```

Comment:
Are you absolutely sure that the minimum order size for Bitstamp BTC-USD pair is 20 BTC? Shouldn't this be the minimum _quote_ amount?
<img width="412" alt="image" src="https://github.com/thrasher-corp/gocryptotrader/assets/9261323/0feb2631-1347-4dcd-810b-e3bdaced2488">

Reply:
changed